install-dev/upgrade/sql/1.7.5.0.sql

```diff
@@ -54,7 +54,7 @@ ALTER TABLE `PREFIX_supply_order_history`
 
 ALTER TABLE `PREFIX_supply_order_receipt_history`
   CHANGE `employee_firstname` `employee_firstname` VARCHAR(255) DEFAULT '',
-  CHANGE `employee_lastname` `employee_firstname` VARCHAR(255) DEFAULT '';
+  CHANGE `employee_lastname` `employee_lastname` VARCHAR(255) DEFAULT '';
```

Comment:
Do we really need to run this command 2 times? (1.7.5.0 & 1.7.5.1)

Reply:
The one in 1.7.5.1 is there so that people who already are in 1.7.5.0 have their database fixed.

Reply:
But for the merchant who haven't upgraded yet, we don't need to apply the same fix 2 times. That's why I would clean this file.

Reply:
Oh so you meant that it should be removed from `1.7.5.0.sql`. Sounds reasonable.